DSA.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <title>Complete RoadMap to DSA</title>
-    <link rel="stylesheet" href=".//css/dsa.css" />
+    <link rel="stylesheet" href="./css/dsa.css" />
```

Comment:
No need of ./ also we can directly use `css/dsa.css`, check how the `style.css` is linked.